src/components/form/file_picker/file_picker.tsx

```diff
@@ -127,6 +128,12 @@ export class EuiFilePicker extends Component<EuiFilePickerProps> {
             ...rest
           } = this.props;
 
+          let UniqueId: string = htmlIdGenerator()();
```

Comment:
```suggestion
          let promptId: string = htmlIdGenerator()();
```

minor nit: we avoid starting variables with `CapitalLetters` unless we `new` them up. (On Kibana this is automated, not sure about EUI.) 

Also, imo, `promptId` is more descriptive than `uniqueId`.